extensions/Article/Entity/Article.xml

```diff
@@ -37,13 +37,6 @@
                description="COM_CONTENT_FIELD_ARTICLETEXT_DESC"
         />
 
-        <field name="author"
-               type="text"
-               label="COM_CONTENT_FIELD_CREATED_BY_ALIAS_LABEL"
-               description="COM_CONTENT_FIELD_CREATED_BY_ALIAS_DESC"
-               default=""
-        />
-
```

Comment:
In our case, the author field is not a text field but a user relation. So it would be a wrong declaration. I mean it can also be that no user extension is installed at all (that's probably never going to happen) and that the articles are flat files with no author.